klasses/bootcamp_admissions_client_test.py

```diff
@@ -91,10 +92,9 @@ def test_happy_path(test_data, mocked_get_200):
     """
     Test BootcampAdmissionClient with a normal response
     """
-    user_email, url = test_data
-    boot_client = BootcampAdmissionClient(user_email)
+    user, url = test_data
+    boot_client = BootcampAdmissionClient(user)
     mocked_get_200.request.assert_called_once_with(url)
-    assert boot_client.admissions == JSON_RESP_OBJ
```

Comment:
Add back as `assert fetch_legacy_admissions(user) == JSON_RESP_OBJ`?

Reply:
That's strange, must have gotten lost in a rebase